storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/put/DefaultPutResolver.java

```diff
@@ -75,14 +82,14 @@ private PutResult insert(@NonNull StorIOSQLiteDb storIOSQLiteDb, @NonNull Conten
     private PutResult updateOrInsert(@NonNull StorIOSQLiteDb storIOSQLiteDb,
                                      @NonNull ContentValues contentValues,
                                      @NonNull String table,
-                                     @NonNull final String idFieldName,
-                                     @NonNull Object id) {
+                                     @NonNull String idFieldName,
+                                     @NonNull String id) {
 
         final int numberOfUpdatedRows = storIOSQLiteDb.internal().update(
                 new UpdateQuery.Builder()
                         .table(table)
                         .where(idFieldName + "=?")
-                        .whereArgs(String.valueOf(id))
```

Comment:
Lol, actually check on line 63 excludes case with `null`, so you are right :+1: 

But I think `idObject` and `idAsString` is more clear, 
If you'll agree, I'll rename `idObject` to `idAsObject`